packages/tabs/src/Tabs.tsx

```diff
@@ -132,36 +133,38 @@ function Tabs({
     setSelected(index);
   }
 
-  function handleKeyDown(e: React.KeyboardEvent) {
-    e.stopPropagation();
+  const getEnabledIndexes: () => [Array<number>, number] = () => {
+    const enabledIndexes: Array<number> = childrenArray.reduce(
```

Comment:
This looks like it should be a `.filter`?

Reply:
It's a filter/map because we need to return the index rather than the child itself. Does this look cleaner?

``` const testIndexes = childrenArray
      .filter(child => !child.props.disabled)
      .map(child => childrenArray.indexOf(child));```

Reply:
Much cleaner. You could even take the map a step further – 

```js
.map(childrenArray.indexOf);
```

I can't think of a case where this indexOf would return -1, so this should be fine unless you can.

Reply:
Get this error: ```Array.prototype.indexOf called on null or undefined

Reply:
And you don't hit that issue when passing in the anonymous function wrapper? Seems strange...

Reply:
It's being given the wrong `this` value in that usage. With `.map(childrenArray.indexOf);` you're just passing in a reference to `indexOf` so when it's called the this value is undefined.